docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
I'm not sure if it will break without the flow type stripping plugin or not. If we decide to, I'll PR separate.